outputs.tf

```diff
@@ -29,20 +29,17 @@ output "uri" {
   description = "The URI of the Pub/Sub topic"
 }
 
-output "subscription_names" {
-  value = "${concat(
-    google_pubsub_subscription.push_subscriptions.*.name,
-    google_pubsub_subscription.pull_subscriptions.*.name
+output "subscriptions" {
```

Comment:
Merging the two groups together means that a collision will occur if a push subscription and a pull subscription share a name. This does not seem desirable. What about providing separate outputs for each category?

Reply:
This was something I thought about before making this particular change. Turns out, push and pull subscriptions share the same namespace in the GCP API so the scenario you are describing is not one that the API will allow. I don't think we need to be concerned with this particular error.